include/wlr/types/wlr_keyboard.h

```diff
@@ -110,6 +112,8 @@ void wlr_keyboard_set_keymap(struct wlr_keyboard *kb,
 void wlr_keyboard_set_repeat_info(struct wlr_keyboard *kb, int32_t rate,
 	int32_t delay);
 void wlr_keyboard_led_update(struct wlr_keyboard *keyboard, uint32_t leds);
+void wlr_keyboard_led_update_from_xkb_state(struct wlr_keyboard *keyboard,
+		struct xkb_state *xkb_state);
```

Comment:
Not a fan of this. Can't we sync the modifiers on all keyboards in the group instead? So that enabling Caps Lock on one keyboard will enable it on all others too?

Reply:
I originally had it that way, but changed it. I don't remember why. I'll look into it again

Reply:
Changed it to sync the modifiers